src/main/cpp/src/row_conversion.cu

```diff
@@ -2270,6 +2276,9 @@ std::unique_ptr<table> convert_from_rows(lists_column_view const &input,
     }
   }
 
+  for (auto &col : output_columns) {
+    col->set_null_count(cudf::null_count(col->view().null_mask(), 0, col->size()));
+  }
```

Comment:
This is not needed, because null mask/count need to be set during column construction in line 2272 above.

Reply:
Yes, sorry. That's not required. I'll remove that.

Reply:
Removed redundant calc.